_layouts/default.html

```diff
@@ -10,7 +10,6 @@
     <main role="main" class="container">
       <section class="jumbotron d-print-none">
         <h1 class="display-3">{{ site.title[page.lang] }}</h1>
-        <h2>{{ site.subtitle[page.lang] }}</h2>
```

Comment:
I would keep the mandate statement on every page. It's what everything we do ties to and is part of the propaganda machine

Reply:
@remyb2canada which part of Mandate statement you would keep on every page? If we have our mandate on every page then we will have to remove <Mandate> submenu , it will be confusing

Reply:
Eventually, Mandate can go to our home(landing page)

Reply:
the mission statement: 
> Enable the strategic value of IT within ESDC by reducing its risks and increasing business agility